src/restic/backend/rest/backend_test.go

```diff
@@ -0,0 +1,87 @@
+// DO NOT EDIT, AUTOMATICALLY GENERATED
```

Comment:
Oh, it is. Here you can find the go generate comment: https://github.com/restic/restic/pull/464/files#diff-6397b881264cb4356775a0e5a3be197fR15

The file is generated from the file https://github.com/restic/restic/blob/master/src/restic/backend/test/tests.go by the program https://github.com/restic/restic/blob/master/src/restic/backend/test/generate_backend_tests.go